src/components/Transaction.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
 
-const Transaction = () => {
+const Transaction = (props) => {
+  const transaction = props.transaction
```

Comment:
You can write this using ES6 shorthand if you like - personal preference but I think it's cool `const {transaction} = props`